thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/detector/anomaly-report-v2.ftl

```diff
@@ -0,0 +1,77 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <table border="0" cellpadding="0" cellspacing="0"
```

Comment:
inline styling everywhere :( internal stylesheets doesn't work for mobile mail apps